### PR TITLE
Describing timeouts' messages

### PR DESCRIPTION
**Pull request by velzevur**
Complementary to aeternity/aeternity#2597

### CODE REVIEW

File: node/api/channels_api_usage.md [hunk 1]
```diff
@@ -1563,7 +1565,53 @@ Examples for this would be either opening amount being below the threshold
...
 
+#### Timeout error
+
+In order to prevent zombie states of the state channel that run indefinitely,
+there are a set of timeouts defined by each specific participant [at WebSocket
+connection opening time](#channel-parameters). Violating those is a violation
+of the off-chain protocol and the non-responsive participant is considered for
+being missing. In this case the connection is interrupted and the client is
+expected either to try reaching for the other party and reconnecting or going
+through the solo closing sequence.
```
Comment (tolbrino) [inline node/api/channels_api_usage.md]: Link to the definition of the sequence would be good.

File: node/api/channels_api_usage.md [hunk 2]
```diff
@@ -1563,7 +1565,53 @@ Examples for this would be either opening amount being below the threshold
 defined by `channel_reserve`, or any of `channel_reserve`, `push_amount` or
 `lock_period` being a negative number.
 
+#### Timeout error
+
+In order to prevent zombie states of the state channel that run indefinitely,
```
Comment (tolbrino) [inline node/api/channels_api_usage.md]: - `that run` => `which run`
- no comma after `indefinitely`

File: node/api/channels_api_usage.md [hunk 3]
```diff
@@ -1563,7 +1565,53 @@ Examples for this would be either opening amount being below the threshold
...
+there are a set of timeouts defined by each specific participant [at WebSocket
```
Comment (tolbrino) [inline node/api/channels_api_usage.md]: `there are...` => `the protocol defines a set of timeouts for each participant`

File: node/api/channels_api_usage.md [hunk 4]
```diff
@@ -1563,7 +1565,53 @@ Examples for this would be either opening amount being below the threshold
...
+connection opening time](#channel-parameters). Violating those is a violation
```
Comment (tolbrino) [inline node/api/channels_api_usage.md]: `Violating those` => `A triggered timeout`

File: node/api/channels_api_usage.md [hunk 5]
```diff
@@ -1563,7 +1565,53 @@ Examples for this would be either opening amount being below the threshold
...
+of the off-chain protocol and the non-responsive participant is considered for
```
Comment (tolbrino) [inline node/api/channels_api_usage.md]: `for being missing` => `to be missing`.

File: node/api/channels_api_usage.md [hunk 6]
```diff
@@ -1563,7 +1565,53 @@ Examples for this would be either opening amount being below the threshold
...
+    "channel_id": "ch_...",
+    "data": {
+      "event": "timeout"
+    }
+  },
+  "version": 1
+}
+```
+
+After it the client receives the following message indicating the closing of
```
Comment (tolbrino) [inline node/api/channels_api_usage.md]: `After it` => `Subsequently`

File: node/api/channels_api_usage.md (hunk 1 above)
Comment (velzevur) [reply node/api/channels_api_usage.md]: But since this is not handled by the FSM, this shall probably refer to the `ON-CHAIN.md`?

Comment (tolbrino) [reply node/api/channels_api_usage.md]: Right.

**Pull request merged**